src/utils/sourceStyles.js

```diff
@@ -4,30 +4,24 @@ export const sourceStyles = {
     minHeight: '300px',
     maxWidth: '100%', // Fill parent before overflowing
     maxHeight: '100%', // Fill parent before overflowing
-    borderRadius: '5px',
```

Comment:
Why were these removed?

Reply:
Inline styles is hard to rewrite, don't if it is not necessary.